tests/test_segments.py

```diff
@@ -709,6 +709,7 @@ def test_date(self):
 		with replace_attr(self.module, 'datetime', Args(now=lambda: Args(strftime=lambda fmt: fmt))):
 			self.assertEqual(self.module.date(pl=pl), [{'contents': '%Y-%m-%d', 'highlight_groups': ['date'], 'divider_highlight_group': None}])
 			self.assertEqual(self.module.date(pl=pl, format='%H:%M', istime=True), [{'contents': '%H:%M', 'highlight_groups': ['time', 'date'], 'divider_highlight_group': 'time:divider'}])
+		self.module.date(pl=pl, format='\u231a')
```

Comment:
This needs to be `self.assertEqual`. It is possible that such change will require using `contents = datetime.now().strftime(format.encode('utf-8')).decode('utf-8')` in `time.py` in place of what is there now as I am not sure that `u'\u231a'` can be compared with string which `date` will return without some unicode error.